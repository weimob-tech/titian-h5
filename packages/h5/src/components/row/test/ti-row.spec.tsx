import { newSpecPage } from '@stencil/core/testing';
import { TiRow } from '..';

describe('ti-row', () => {
  it('renders', async () => {
    const page = await newSpecPage({
      components: [TiRow],
      html: `<ti-row></ti-row>`,
    });
    expect(page.root).toMatchSnapshot();
  });

  it('renders gutter success', async () => {
    const page = await newSpecPage({
      components: [TiRow],
      html: `<ti-row gutter="16"></ti-row>`,
    });
    expect(page.root).toMatchSnapshot();
  });
});
