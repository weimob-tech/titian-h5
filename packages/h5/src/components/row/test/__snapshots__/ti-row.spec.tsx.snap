// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ti-row renders 1`] = `
<ti-row>
  <mock:shadow-root>
    <div class="titian-row titian-row-flex" style="margin-left: -0px; margin-right: -0px;">
      <slot></slot>
    </div>
  </mock:shadow-root>
</ti-row>
`;

exports[`ti-row renders gutter success 1`] = `
<ti-row gutter="16">
  <mock:shadow-root>
    <div class="titian-row titian-row-flex" style="margin-left: -8px; margin-right: -8px;">
      <slot></slot>
    </div>
  </mock:shadow-root>
</ti-row>
`;
